### DISCUSSION TITLE
java.lang.IllegalArgumentException: Invalid Payment Intent client secret: **********

### DISCUSSION THREAD

**Discussion by navaghandabhi** [flutter-stripe/flutter_stripe#1711] [Q&A]
java.lang.IllegalArgumentException: Invalid Payment Intent client secret: 

I got this error when i call 

```
  await Stripe.instance.retrievePaymentIntent("$_secretKey").then((value) {
        log("PaymentIntent: $value");
      });
```
my secret key is correct ,its working fine to create intent

**Comment by antonio-nicolau** [comment]
> java.lang.IllegalArgumentException: Invalid Payment Intent client secret: 
> 
> 
> 
> I got this error when i call 
> 
> 
> 
> ```
> 
>   await Stripe.instance.retrievePaymentIntent("$_secretKey").then((value) {
> 
>         log("PaymentIntent: $value");
> 
>       });
> 
> ```
> 
> my secret key is correct ,its working fine to create intent 

Hi bro, did you manage to fix that? I am facing the same issue

**Reply by navaghandabhi** [reply]
not yet
